pkg/admission/webhook/generic_webhook.go

```diff
@@ -96,7 +96,7 @@ func (p *WebhookDispatcher) getAPIBindingWorkspace(attr admission.Attributes, cl
 	for _, obj := range objs {
 		apiBinding := obj.(*apisv1alpha1.APIBinding)
 		for _, br := range apiBinding.Status.BoundResources {
-			if apiBinding.Status.BoundAPIExport.Workspace == nil {
+			if apiBinding.Status.BoundAPIExport != nil && apiBinding.Status.BoundAPIExport.Workspace == nil {
```

Comment:
I thought OpenAPI guarantees this?

Reply:
It's optional, a pointer, omitempty. I don't see anything that does. Not in the -patch

Reply:
You are right. What I meant is about the spec reference, not the status:
```
- op: add
  path: /spec/versions/name=v1alpha1/schema/openAPIV3Schema/properties/spec/properties/reference/oneOf
  value:
  - required: ["workspace"]
```